packages/dev/docs/pages/releases/2022-10-03.mdx

```diff
@@ -0,0 +1,299 @@
+{/* Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {BlogPostLayout, Hero} from '@react-spectrum/docs';
+export default BlogPostLayout;
+
+---
+description: tbd
+date: 2022-10-03
+---
+
+# October 3, 2022 Release
+We have been busy this month with new components and plenty of drag and drop updates. This release we moved drag and drop to RC which comes  with some changes to our API as well as introducing a new set of higher level drag and drop hooks to make building common experiences even easier. See our [update](#drag-and-drop-updates) on drag and drop below for more information. Our new components are [Badge](https://react-spectrum.adobe.com/react-spectrum/Badge.html) and [LabeledValue](https://react-spectrum.adobe.com/react-spectrum/LabeledValue.html). Badges are handy for showing a small amount of color-categorized metadata and LabeledValue is for displaying a non-editable values with a label such as a properties panel. It also formats numbers, dates, times, and lists according to the user's locale.
```

Comment:
```suggestion
We have been busy this month with new components and plenty of drag and drop updates. This release we moved drag and drop to RC which comes  with some changes to our API as well as introducing a new set of higher level drag and drop hooks to make building common experiences even easier. See our [update](#drag-and-drop-updates) on drag and drop below for more information. Our new components are [Badge](https://react-spectrum.adobe.com/react-spectrum/Badge.html) and [LabeledValue](https://react-spectrum.adobe.com/react-spectrum/LabeledValue.html). Badges are handy for showing a small amount of color-categorized metadata and LabeledValue is for displaying a non-editable values with a label such as in a properties panel. It also formats numbers, dates, times, and lists according to the user's locale.
```

Reply:
also, extra space in `RC which  comes with`